### ISSUE TITLE
Lapis is sold as inksacks

### ISSUE THREAD

**Issue by s1h4d0w** [EssentialsX/Essentials#2703]
### Information

**Full output of `/ess version`**:

```
[14:07:42 INFO]: Server version: 1.12.2-R0.1-SNAPSHOT git-Spigot-79a30d7-acbc348 (MC: 1.12.2)
[14:07:42 INFO]: EssentialsX version: 2.17.0.5
[14:07:42 INFO]: PermissionsEx version: 1.23.4
[14:07:42 INFO]: Vault version: 1.7.2-b107
[14:07:42 INFO]: EssentialsXSpawn version: 2.17.0.5
[14:07:42 INFO]: You are running unsupported plugins!
```

**Server log**: https://gist.github.com/s1h4d0w/38b4e909842c48e774cd9e487cc3c41a

**EssentialsX config**: https://gist.github.com/s1h4d0w/48284a730e1871de11e75f6618ba3153

**EssentialsX worth.yml:** https://gist.github.com/s1h4d0w/ba421f3fe80a8aeb6371517315e45173

### Details

**Description**  
When using `/sell hand` while holding lapis, the item will be sold as an inksack. When using `/shop` and navigating to the corresponding shop that buys lapis it will also show as an inksack, with the lapis sprite. **However** if the player goes to the physical shop they will be able to buy lapis like normal.

If I do `/give <user> lapis` then that user receives lapis, but the chat says it's giving the player ink sack.

**Steps to reproduce**  
Sell lapis using `/sell hand` or give lapis using `/give <user> lapis`

**Expected behavior**  
The user being able to sell lapis as lapis, and get the proper price for them

**Screenshots**  
![image](https://user-images.githubusercontent.com/5654654/62210694-0ebcc380-b39d-11e9-954c-e6955b076522.png)
How the lapis shop is shown in the shop plugin

![image](https://user-images.githubusercontent.com/5654654/62210795-46c40680-b39d-11e9-9fa2-9e6ee8fb226c.png)
Chat output from `/give lepel lapis` and `/sell hand`

**Comment by s1h4d0w** [comment]
Adding to this, my plugin list:

```
- KeepItems
- ArmoredElytra
- ConsoleSpamFix
- ChatReaction
- SkinsRestorer
- AutoRestart
- CoreProtect
- ClearLag
- ChatControl
- Genie
- Votifier
- ProtocolLib
- Multiverse-Core
- LibsDisguises
- FastAsyncWorldEdit
- commandDelay
- PermissionsEx
- AntiBuild
- ServerListPlus
- Vault
- PlayerMarkers
- UltraCosmetics
- Parties
- Matrix
- PerWorldInventory
- HolographicDisplays
- HeadsPluginAPI
- PlayerEmotesPro
- MyCommand
- InfernalMobs
- WorldEdit
- NoSwear
- AdvancementMoney
- Essentials
- AutoMessager
- OnTime
- SuperbVote
- WorldEditSelectionVisualizer
- TouchscreenHolograms
- HeadsInventory
- WorldGuard
- MobStacker
- Shop
- BetterRTP
- EssentialsSpawn
- WorldGuardExtraFlags
- SilkSpawners
- DenyTP
- ChopTree3
- GriefPrevention
- AuthMe
- CrazyEnchantments
```

**Comment by AMDBartek** [comment]
I can confirm that this is a bug and not only Lapis gets sold as inksacks, it's almost every dye that is in Minecraft. This is because every dye including Lapis is `minecraft:dye` but with an extra numerical value. You can test this by executing `/minecraft:give [Username] minecraft:dye 1 4` this command will give you Lapis and `/minecraft:give [Username] minecraft:dye 1 0` will give you inksacs so I think the cause of this issue is EssentialsX not reading the extra numerical value that the item has.

**Comment by pop4959** [comment]
This should be fixed on MineCraft 1.13+ by the sheer fact that lapis is now it's own item, rather than being lumped with the other dyes (which stem from ink sack). I don't believe this will ever be fixed in prior versions, since it hasn't already been fixed until now.

**Comment by mdcfe** [comment]
To clarify: on older versions of Minecraft, EssentialsX will use the value you set for each data value of each material in `worth.yml`. On 1.13+, EssentialsX will use the value you set for that material, as data values don't exist any more.
